pandas/tests/test_lib.py

```diff
@@ -39,68 +39,359 @@ def test_infer_dtype_bytes(self):
         arr = arr.astype(object)
         self.assertEqual(pd.lib.infer_dtype(arr), compare)
 
+    def test_maybe_indices_to_slice_left_edge(self):
+        target = np.arange(100)
 
-class TestIsscalar(tm.TestCase):
+        # not slice
+        indices = np.array([], dtype=np.int64)
```

Comment:
Correct, fixed the test case to use arange (to be representable by slice) and loops.